### DISCUSSION TITLE
Multiple labels in one job

### DISCUSSION THREAD

**Discussion by sarkis1997** [apenab/react-dymo#33] [Q&A] [answered]
Hi, is there also any possibility to print multiple labels in one job. In the original documentation there is something like dymo.label.framework.LabelSetBuilder. How can I achieve this with the dymo-react-hooks?

**Comment by apenab** [comment]
@sarkis1997 can you share the link of the documentation? I don't find the Dymo doc.

**Reply by sarkis1997** [reply]
@apenab [LabelSetBuilder](http://labelwriter.com/software/dls/sdk/docs/DYMOLabelFrameworkJavaScriptHelp/symbols/dymo.label.framework.LabelSetBuilder.html)

[documentation](https://github.com/dymosoftware/dymo-connect-framework)

**Reply by apenab** [reply]
Ok, I think it's a nice new feature to implement. Feel free to open a new issue with the "new feature" tag, I'll implement ASAP

**Comment by apenab** [comment, marked as answer]
@sarkis1997 Although we can implement a new utility function to achieve this feature request, I think that this solution could work:

```js

const dymoParams = [params1, params2]
const dymoRequests = []

dymoParams.forEach((params) => {
    dymoRequests.push(dymoRequestBuilder({
            method: "POST",
            wsAction: "printLabel",
            axiosOtherParams: params,
        }))
})

Promise.all(dymoRequests).then((values) => {
  console.log(values);
});

```

**Reply by sarkis1997** [reply]
It didn't do the job, because the labelprinter still receives seperate requests. I think the param `&labelSetXml=` should be used instead of `&labelXml=` to achieve this.
